tests/test_assets.py

```diff
@@ -1376,6 +1450,538 @@ def test_error_message_plurality(self,
+                    )
+                    assert_equal(actual_asset, assets[n])
+                    assert_equal(
+                        actual_asset.exchange_info.country_code,
+                        self.country_code(n),
+                    )
+
+    def test_lookup_symbol_fuzzy(self):
+        num_exchanges = 3
+        metadata = pd.DataFrame.from_records([
+            {'symbol': symbol, 'exchange': 'EXCHANGE %d' % n}
+            for n in range(num_exchanges)
+            for symbol in ('PRTY_HRD', 'BRKA', 'BRK_A')
+        ])
+        exchanges = pd.DataFrame({
+            'exchange': metadata['exchange'].unique(),
+            'country_code': list(map(self.country_code, range(num_exchanges))),
+        })
+        self.write_assets(equities=metadata, exchanges=exchanges)
+        finder = self.asset_finder
+        dt = pd.Timestamp('2013-01-01', tz='UTC')
+
+        # Try combos of looking up PRTYHRD with and without a time or fuzzy
+        # Both non-fuzzys get no result
+        with self.assertRaises(SymbolNotFound):
+            finder.lookup_symbol('PRTYHRD', None)
+        with self.assertRaises(SymbolNotFound):
+            finder.lookup_symbol('PRTYHRD', dt)
+
+        for n in range(num_exchanges):
+            with self.assertRaises(SymbolNotFound):
+                finder.lookup_symbol(
+                    'PRTYHRD',
+                    None,
+                    country_code=self.country_code(n),
+                )
+                with self.assertRaises(SymbolNotFound):
+                    finder.lookup_symbol(
+                        'PRTYHRD',
+                        dt,
+                        country_code=self.country_code(n),
+                    )
+
+        with self.assertRaises(MultipleSymbolsFound):
+            finder.lookup_symbol('PRTYHRD', None, fuzzy=True)
+
+        with self.assertRaises(MultipleSymbolsFound):
+            finder.lookup_symbol('PRTYHRD', dt, fuzzy=True)
+
+        def check_sid(expected_sid, ticker, n):
+            params = (
+                {'as_of_date': None},
+                {'as_of_date': dt},
+                {'as_of_date': None, 'fuzzy': True},
+                {'as_of_date': dt, 'fuzzy': True},
+            )
+            for extra_params in params:
+                with self.assertRaises(MultipleSymbolsFound):
+                    finder.lookup_symbol(ticker, **extra_params)
+
+                self.assertEqual(
+                    expected_sid,
+                    finder.lookup_symbol(
+                        ticker,
+                        country_code=self.country_code(n),
+                        **extra_params
+                    ),
+                )
+
+        for n in range(num_exchanges):
+            check_sid(n * 3, 'PRTY_HRD', n)
+            check_sid(n * 3 + 1, 'BRKA', n)
+            check_sid(n * 3 + 2, 'BRK_A', n)
+
+    def test_lookup_symbol_change_ticker(self):
+        T = partial(pd.Timestamp, tz='utc')
+        num_exchanges = 3
```

Comment:
One thing that I've found a bit confusing while reading through these tests is that they're generally written with reference to the number of exchanges, but it seems like for the most part the thing we actually care about is the country code..

Reply:
yeah, I should rename this